docs/setup/appstore.md

```diff
@@ -0,0 +1,26 @@
+# App Store
+
+**Setup :material-menu-right: App Store**
+
+The **ConnexCS App Store** lets you discover, download, and install applications on your user account.
+
+The Apps created using the IDE when published and are available for use in the App Store.
```

Comment:
The Apps created using the IDE are available for use in the App Store when published.

Reply:
done